src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { ButtonHTMLAttributes, MouseEventHandler, ReactNode } from 'react';
+
+// Styles
+import './button.css';
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: 'primary' | 'secondary' | 'tertiary';
+  children?: ReactNode;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+const Button = ({
```

Comment:
We can use Button.defaultProps = {
variant = 'primary',
  children = '',
  onClick = () => {}
}